engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorData.scala

```diff
@@ -19,17 +22,25 @@ final case class WorkflowExecutionDiff(executionStore: Map[JobKey, ExecutionStat
 
 case class WorkflowExecutionActorData(workflowDescriptor: EngineWorkflowDescriptor,
                                       executionStore: ExecutionStore,
-                                      outputStore: OutputStore) extends WdlLookup {
+                                      outputStore: OutputStore,
+                                      serviceRegistryActor: ActorRef) extends WdlLookup {
 
   override val expressionLanguageFunctions = new WdlFunctions(workflowDescriptor.backendDescriptor.workflowOptions)
 
-  def jobExecutionSuccess(jobKey: JobKey, outputs: CallOutputs) = this.copy(
-    executionStore = executionStore.add(Map(jobKey -> Done)),
-    outputStore = outputStore.add(updateSymbolStoreEntry(jobKey, outputs))
-  )
+  def jobExecutionSuccess(jobKey: JobKey, outputs: JobOutputs) = {
+
+    val metadataKey = MetadataKey(workflowDescriptor.id, Some(MetadataJobKey(jobKey.scope.fullyQualifiedName, jobKey.index, jobKey.attempt)), CallMetadataKeys.ExecutionStatus)
```

Comment:
I'll make the changes but I'm curious about how strongly people really feel about this -

I personally prefer the readability of `Some(x)` over `Option(x)`, given that `x` in this case is definitely not a null value and `Some` is closer to my intention.